scrapy/downloadermiddlewares/retry.py

```diff
@@ -70,8 +72,10 @@ def _retry(self, request, reason, spider):
             retryreq.meta['retry_times'] = retries
             retryreq.dont_filter = True
             retryreq.priority = request.priority + self.priority_adjust
+            self.stats.inc_value('retry/count')
```

Comment:
I thought that it could be redundant - you'll get the same exception names as in downloader, e.g. `'downloader/exception_type_count/twisted.internet.error.ConnectionRefusedError': 155` and `'retry/twisted.internet.error.ConnectionRefusedError': 155`; `downloader/response_status_count/` will be also duplicated - it could be 20x lines in stats with very little new information.